Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -58,6 +58,64 @@ class MongoSwiftTestCase: XCTestCase {
     }
 }
 
+extension RuntimeError: Equatable {
+    public static func == (lhs: RuntimeError, rhs: RuntimeError) -> Bool {
+        switch (lhs, rhs) {
+        case (.internalError(message: _), .internalError(message: _)),
```

Comment:
Does this just equate them if they generically match `internalError`, regardless of message and/or code?